public/style.css

```
@@ -1971,6 +1975,11 @@ video {
     font-size: 1.25rem;
     line-height: 1.75rem;
   }
+
+  .md\:text-lg{
```

```suggestion
  .md:text-lg{
```
or maybe this is some CSS thing I don't know?

---

This file is generated from tailwind

---

neat, must be some css thing I'm ignorant of :)